mitielib/include/mitie.h

```diff
@@ -151,6 +151,13 @@ extern "C"
             - The returned object MUST BE FREED by a call to mitie_free().
             - If the object can't be created then this function returns NULL.
     !*/
+    typedef struct mitie_total_word_feature_extractor mitie_total_word_feature_extractor;
+    MITIE_EXPORT mitie_named_entity_extractor* mitie_load_named_entity_extractor_pure_model_without_feature_extractor (
+        const char* filename
+    );
+    /*!
+	Added for Rasa_nlu multitenancy
```

Comment:
What does this mean?

More generally, all the functions in this file should be properly documented.  e.g. what does this function do?  What are the requirements for calling it?